src/mlpack/core/cv/metrics/top_k_accuracy_impl.hpp

```diff
@@ -19,6 +19,7 @@ template<typename MLAlgorithm, typename DataType, typename TopK>
 double TopKAccuracy::Evaluate(MLAlgorithm& model,
                         const DataType& data,
                         const arma::Row<size_t> labels,
+                        const arma::Row<size_t> predsprob
```

Comment:
Looks like this is only needed for testing? You can use `model.Probabilities()` to the probabilities. That way this still work with the hyperparameter framework.

Reply:
Sorry for the delay. Oh, I got your point, made the changes, will it return the probabilities of all the classes or the final reduced_sum probability?
Thanks